build.py

```diff
@@ -6,6 +6,9 @@
 import subprocess
 
 is_travis = os.environ.get('TRAVIS') is not None
+
+engine = os.environ.get('OCTOBOT_CONTAINER_BUILD_ENGINE', 'docker')
```

Comment:
imo, a cli arg that explicitly chooses podman instead feels better than an env variable turning into shell execution. thoughts?